src/components/NavBar.css

```diff
@@ -1,42 +1,65 @@
-* {
-  box-sizing: border-box;
-}
-
 .menuNav {
   display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-  background-color: rgb(104, 103, 143);
-  width: 100%;
+  justify-items: center;
+  align-items: center;
+  justify-content: flex-start;
+  width: 70%;
+  height: 5.938rem;
+  background-color: #fff;
   margin-left: auto;
   margin-right: auto;
-  border-bottom-left-radius: 50px;
-  border-bottom-right-radius: 50px;
+  padding-left: 6.17rem;
+  padding-right: 6.25rem;
+  margin-top: 1rem;
+  border-bottom: #e8e8e8 solid;
+}
+
```

Comment:
- [ ] Kindly make sure your header/menuNav sticks as much as possible to the one from the **[template design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)**(e.g.: add elements on the header, use correct text_colors, spacing between and around elements, fonts(family, size. weight,...)...).
### Template design
![Screenshot from 2022-11-11 17-52-14](https://user-images.githubusercontent.com/83042742/201381761-7713d1f1-79ff-436f-81e3-fcd4d1348d6d.png)

### Your design
![Screenshot from 2022-11-11 18-02-13](https://user-images.githubusercontent.com/83042742/201381678-f463d52a-3c88-46e1-8da3-d7d678eb5f2f.png)